pybrake/django.py

```diff
@@ -68,6 +108,8 @@ def process_view(self, request, view_func, view_args, view_kwargs):
             route += "." + view_func.__name__
             metric.route = route
 
+        request = None
```

Comment:
add a comment what this code does

Reply:
I tried to set the mutable object to None, but that's not how it works.